netlify.toml

```diff
@@ -0,0 +1,19 @@
+# This should be used for docs also
+[[headers]]
+  for = "/*"
+  [headers.values]
+    Content-Security-Policy = "default-src https:; script-src https: 'unsafe-inline'; style-src https: 'unsafe-inline'; img-src https: data: blob:"
+    Permissions-Policy = "geolocation=(),midi=(),sync-xhr=(self),microphone=(),camera=(),magnetometer=(),gyroscope=(),fullscreen=(self),payment=()"
+    Referrer-Policy = "no-referrer"
+    X-Frame-Options = "SAMEORIGIN"
+    X-Xss-Protection = "1; mode=block"
```

Comment:
Looks like this has been deprecated: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/X-XSS-Protection

Reply:
This came from https://docs.netlify.com/routing/headers/#syntax-for-the-headers-file

Reply:
https://securityheaders.com/?q=ui.rilldata.io&followRedirects=on this doesn't require this so we can remove it.